chronosphere/notification_policy_data.go

```diff
@@ -52,6 +52,7 @@ type Routes struct {
 type RoutesNotifierList struct {
 	Notifiers      []*RoutesNotifierListNotifier `json:"notifiers"`
 	RepeatInterval string                        `json:"repeat_interval,omitempty"`
+	GroupBy        *RoutesNotifierListGroupBy    `json:"group_by,omitempty"`
```

Comment:
nit: comment on this struct could benefit from an update now that it includes GroupBy in addition to repeat interval + list of notifiers